azure-pipelines.release.yml

```diff
@@ -75,9 +66,10 @@ steps:
       npm run publish:beachball -- -b origin/6.0 -t lts-6 -n $(npmToken) --access public -y
     displayName: 'Publish Change Requests and Bump Versions'
 
-  - script: |
-      node scripts/updateReleaseNotes.js --token=$(githubPAT) --apply --debug
-    displayName: 'Update github release notes'
+  # TODO: updateReleaseNotes script results in too many calls to the github API causing a timeout, needs to be refactored
+  # - script: |
+  #     node scripts/updateReleaseNotes.js --token=$(githubPAT) --apply --debug
```

Comment:
This script to update the github release notes is causing a github API limit [error](https://uifabric.visualstudio.com/UI%20Fabric/_build/results?buildId=279592&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=c3091b1e-314e-512e-11db-151fdba14cd6) in the pipeline runs so commenting out